examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+                                      method='shrunk')
+    # Read forward model
+    forward = mne.read_forward_solution(fname_fwd)
+
+    # Compute weights of free orientation (vector) beamformer with weight
+    # normalization (neural activity index, NAI). Providing a noise covariance
+    # matrix enables whitening of the data and forward solution. Source
+    # orientation is optimized by setting pick_ori to 'max-power'.
+    # weight_norm can also be set to 'unit-noise-gain'. Source orientation can
+    # also be 'normal' (but only when using a surface-based source space) or
+    # None, which computes a vector beamfomer. Note, however, that not all
+    # combinations of orientation selection and weight normalization are
+    # implemented yet.
+    filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                        noise_cov=noise_cov, pick_ori='max-power',
+                        weight_norm='nai')
+
+    # Apply this spatial filter to the evoked data.
+    stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+    # take absolute values for plotting
+    stc.data[:, :] = np.abs(stc.data)
+
+    # Save result in stc files if desired
+    if fname is not None:
+        stc.save('lcmv-vol')
+
+    # select time window (tmin, tmax) in ms - consider changing for real data
+    # scenario, since those values were chosen to optimize computation time
+    stc.crop(0.087, 0.087)
+
+    src = forward['src']
+
+    # Save result in a 4D nifti file
+    img = mne.save_stc_as_volume(fname, stc, src,
+                                 mri_resolution=True)
+
+    return img, stc, src
+
+
+###############################################################################
+# Save non linear mapping data
+
+def write_morph(fname, morph, overwrite=True):
+    morph_out = dict()
+
+    # dissolve object structure
+    for key, value in morph.items():
+        # save type for order independent decomposition
+        if hasattr(value, '__dict__'):
+            value = value.__dict__
+        morph_out[key] = value
+    if not fname.endswith('.h5'):
+        fname += '.h5'
+    write_hdf5(fname, morph_out, overwrite=overwrite)
+
+
+###############################################################################
+# Load non linear mapping data
+
+def read_morph(fname):
+    if not fname.endswith('.h5'):
+        fname += '.h5'
+    morph_in = read_hdf5(fname)
+
+    morph = dict()
+    # create new instances
+    morph['mapping'] = imwarp.DiffeomorphicMap(None, [])
+    morph['mapping'].__dict__ = morph_in.get('mapping')
+    morph['affine'] = imaffine.AffineMap(None)
+    morph['affine'].__dict__ = morph_in.get('affine')
+
+    morph['affine_reg'] = morph_in.get('affine_reg')
+    morph['domain_shape'] = morph_in.get('domain_shape')
+
+    return morph
+
+
+###############################################################################
+# Execute example
+
+# Settings
+voxel_size = (5., 5., 5.)  # of the destination volume
```

Comment:
This comment seems misleading. `voxel_size` is used for resampling both src and dest volumes for the purposes of making them uniformly sampled or (in this case) making the operation faster, right?

Reply:
Well, yes... but in the end I would like to use two independent ``voxel_size``s (that should be called differently, I agree): one for the voxel size of the volume the user cares about (lcmv data) and one that is used to compute the morph (kind of like a measure for precision)... I would then reslice the anatomicals to that size, compute the morph, reslice the lcmv data to the same space, morph it and then reslice it to the target space (which should have lower or equal resolution than the space used to compute the morph).